renderlib/graphics/ImageXyzcGpu.cpp

```diff
@@ -87,7 +95,12 @@ ImageGpu::createVolumeTextureFusedRGBA8(ImageXYZC* img)
 void
 ImageGpu::createVolumeTexture4x16(ImageXYZC* img)
 {
-  m_gpuBytes += (16 * 4) / 8 * img->sizeX() * img->sizeY() * img->sizeZ();
+  int N = 4;
+  if (img->sizeC() < 4) {
+    N = img->sizeC();
+  }
```

Comment:
Equivalent to doing `N = std::min(4, img->sizeC())`, right?
(I guess not worth replacing since you'd need to include `<algorithm>`)